.storybook/main.js

```diff
@@ -29,24 +27,21 @@ module.exports = {
     "@storybook/addon-controls",
     "@storybook/addon-queryparams",
   ],
-  // We want the repo's webpack configuration to be a part of the Storybook
-  // configuration for SCSS and SVGs, but we want to keep the existing
-  // configuration loaders to use MDX when the project builds.
-  webpackFinal: config => {
-    const { rules } = config.module;
-    // We still want MDX-related configuration rules from Storybook's webpack
-    // which come from `rules[4]`. This is not ideal as the order of the
-    // rules can change and this will break if this rule is removed.
-    // The custome webpack rules added here are from `webpack.config.js` and
-    // they are loaders to handle SVG loading, Typescript compilation, and
-    // CSS generation from SCSS.
-    const customRules = [rules[4], ...custom.module.rules];
-    return {
-      ...config,
-      module: {
-        ...config.module,
-        rules: customRules,
-      },
-    };
+  typescript: {
+    // Type-check stories during Storybook build.
+    check: true,
+  },
+  webpackFinal: async (config, { configType }) => {
```

Comment:
So because Storybook is the way it is, it's still being built with webpack while the app is being built with rollup, is that correct?  Is there a danger of running into a problem where stuff is going to work in storybook but not in a consuming app?

Reply:
Yeah, that's the case. I don't think so -- both webpack and rollup transpile/compile the code and minify it. You can still choose how to compile/minify and how to export it. The final output of both, however, is basically the same. I haven't seen any differences between what webpack and rollup produce but that's a good thing to keep an eye on.